developer/src/common/web/utils/src/types/ldml-keyboard/ldml-keyboard-xml-reader.ts

```diff
@@ -241,6 +262,9 @@ export class LDMLKeyboardXMLSourceFileReader {
         // mark all children as an implied import
         subsubval.forEach(o => o[ImportStatus.impliedImport] = basePath);
       }
+      if (!base) {
```

Comment:
Let's use the same comparison as above for consistency

```suggestion
      if (base !== constants.cldr_import_base) {
```